iroha_data_model/src/lib.rs

```diff
@@ -1334,6 +1353,20 @@ pub mod transaction {
         pub fn sign(self, key_pair: &KeyPair) -> Result<VersionedTransaction> {
             self.into_inner_v1().sign(key_pair).map(Into::into)
         }
+
+        /// Returns time of transaction creation
+        pub const fn creation_time(&self) -> u64 {
```

Comment:
There is a method for other versioned transactions `.payload()` it does the same thing but returns just payload. May be it would be more suitable to have something like this here and have only 1 method?

Reply:
Could not find such method but add them and removed methods `creation_time` and `creator`